src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
```suggestion
public class ColorSupplier {
    private final Random random = new Random();
    
    public String getRandomColor() {
```
lets create an instance of Random as class field - to avoid creating a new instance each time the method is called

Reply:
Corrected